eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -47,6 +47,12 @@
         <Bug pattern="UUF_UNUSED_PUBLIC_OR_PROTECTED_FIELD"/>
     </Match>
 
+    <Match>
+      <Class name="com.microsoft.azure.eventhubs.impl.EventHubClientImpl"/>
+      <Field name="USER_AGENT"/>
+      <Bug pattern="MS_SHOULD_BE_FINAL"/>
+    </Match>
```

Comment:
Note that every other match statement has a comment describing why it exists. Please ensure you also add these to explain why the suppression is necessary.